README.md

```diff
@@ -11,6 +17,9 @@ Launch Packrat:
 
     docker-compose up
 
+You may need access to HackEDU's Bootstrap modules in a private repository for
+this to run correctly. Ask the maintainers for access if you need this.
```

Comment:
Can you change this wording to `You may need access to the Bootstrap theme we bought for Packrat, which is in a private submodule. Please create an issue if you need access and we'll work with you to make that happen.`.